examples/launcher-templates/default/2.3.0/job.tpl

```diff
@@ -0,0 +1,213 @@
+# Version: 2.3.0
```

Comment:
The only thing that gives me pause is the version diff between these launcher templates and launcher - I could imagine a world where we decide to call these the `2.9.0` templates, given they correspond to the recent `2.9.0` launcher release.

I could see why we'd version them separately though, these templates don't necessarily have to match a specific launcher release nor have they explicitly in the past.

Reply:
We're not inventing this versioning scheme; these are the versions provided by the launcher.

Reply:
Ah I see, ty for that context - looks like we control this versioning at https://github.com/rstudio/launcher/blob/main/src/cpp/job_launcher/impls/kubernetes/KubernetesTemplating.hpp#L30-L31.
Maybe something to discuss with my team in the `2.10` release then, let's resolve this thread in the meantime.